manifests/r/RescueTime,Inc/RescueTime/2.16.8.2/RescueTime,Inc.RescueTime.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RescueTime,Inc.RescueTime
+PackageVersion: 2.16.8.2
+Installers:
+- Architecture: x86
+  InstallerType: inno
+  InstallerUrl: https://www.rescuetime.com/installers/RescueTimeInstaller.exe
+  InstallerSha256: A6CFC215218B562CBE4FD37178D03C2F811E230C4C01AB1B0D6B163323117455

```

Product code should be included for application matching since the DisplayVersion is empty; AppsAndFeaturesEntries should be included since the version is in the DisplayName
```suggestion
  InstallerSha256: A6CFC215218B562CBE4FD37178D03C2F811E230C4C01AB1B0D6B163323117455
  ProductCode: '{2505571C-03B3-4F9F-AC35-33F1CB4B5E9E}_is1'
  AppsAndFeaturesEntries:
  - DisplayName: RescueTime 2.16.8.2
    DisplayVersion: null
    Publisher: RescueTime.com
    ProductCode: '{2505571C-03B3-4F9F-AC35-33F1CB4B5E9E}_is1'
```